pom.xml

```
@@ -23,7 +23,7 @@
     <maven.skip.it.tests>false</maven.skip.it.tests>
     <version.com.fasterxml.jackson.core.jackson-databind>2.13.2.2</version.com.fasterxml.jackson.core.jackson-databind>
     <version.frontend.plugin>1.12.0</version.frontend.plugin>
-    <version.io.quarkus>2.13.7.Final</version.io.quarkus>
+    <version.io.quarkus>2.16.7.Final</version.io.quarkus>
```

the only possible upgrade is 2.13.8.Final
```suggestion
    <version.io.quarkus>2.13.8.Final</version.io.quarkus>
```